### PR TITLE
Add Values and Timeline section

### PR DESCRIPTION
**Pull request by aayush-05**
--Added new Values and Timeline Section (data to be added/provided by @jaskiratsingh2000)
--Made everything more responsive and better looking on even smaller displays

### CODE REVIEW

**Comment by aayush-05** [comment]
@jaskiratsingh2000 @vaibhavdaren please review

File: src/pages/Team/TeamMembers.js [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+      "https://avatars3.githubusercontent.com/u/31003923?s=460&u=c2c911e24cc6533df2048b20f35a9714e2227e83&v=4",
+    github: "https://github.com/AuraOfDivinity",
+    linkedin: "https://www.linkedin.com/in/asel-peiris-76536158/",
+  },
+  {
+    name: "Saurabh Raj",
+    post: "Maintainer",
+    headline: "CommunityBridge Mentee at Codeuino | MERN developer",
+    profile:
+      "https://avatars1.githubusercontent.com/u/39027928?s=460&u=e9678c517c8ba536b5f8bba195dc5ea33b637ed8&v=4",
```
Comment (ksraj123) [inline src/pages/Team/TeamMembers.js:38]: All the other images in this section are rounded rectangles, mine is circular, you can use my image from [here](https://drive.google.com/file/d/1EVjX8K7PVBensQTUPLC3KRrmqslARNtO/view?usp=sharing) Also, my name is "Kumar Saurabh Raj", you can abbreviate it as "Kr. Saurabh Raj" if that's too long. :smiley:

**Review by ksraj123 — COMMENTED** [review]
Good Work!, But some things could be improved.

Please go through my comments. Thanks

File: src/components/NavBar/index.js [hunk 2]
```diff
@@ -1,140 +1,225 @@
...
 import { Nav, Navbar, NavDropdown, Image } from "react-bootstrap";
 import logo from "../../logo.png";
 
-const NavBar = () => {
-  const navbarRef = useRef(null);
+class NavBar extends React.Component {
+  state = {
+    firstIsOpen: false,
+    secondIsOpen: false,
+    thirdIsOpen: false,
```
[the remark this answers is not in the document]
Comment (aayush-05) [reply src/components/NavBar/index.js:13]: The problem with having one state property is that then all four dropdowns will be referring to the same state, making all of them toggle even when one is hovered. 
External libraries don't offer show on hover functionality afaik. Please suggest any if you know

[the remark this answers is not in the document]
Comment (ksraj123) [reply src/components/NavBar/index.js:13]: No, what I meant was that you can do this have a single property for all four dropdowns which is null initially and when one is hovered then you can set it to "1" or something else whose value you can check for to expand dropdown 1, it does not have to be boolean type.

File: src/pages/Home/Components/Statistics.js [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+        .data.length,
+    });
+  };
+
+  getForksAndStars = async () => {
+    let totalStars = 0;
+    let totalForks = 0;
+    await axios
+      .get("https://api.github.com/orgs/codeuino/repos")
+      .then((response) => {
```
[the remark this answers is not in the document]
Comment (aayush-05) [reply src/pages/Home/Components/Statistics.js:26]: My bad

File: src/components/NavBar/index.js (hunk 2 above)
[the remark this answers is not in the document]
Comment (aayush-05) [reply src/components/NavBar/index.js:13]: The problem I am facing is that a dropdown is made visible using the show attribute (accepts either a true or false value). Now if we would be having a single state attribute referenced by all dropdowns in their show attributes, then changing that single attribute would toggle all dropdowns on re render even when the cursor is hovered on one. Any way out?
image
Other than this, rest all issues are fixed. Please review

[the remark this answers is not in the document]
Comment (ksraj123) [reply src/components/NavBar/index.js:13]: I have opened a [PR](https://github.com/aayush-05/website-www.codeuino.org/pull/1) to your repo having changes we wanted. Merging it should reflect the changes here in this PR or you could make those changes yourself.

Here's how the code would look like,
```
  state = {
    navDrop: null
  };

  handleToggle = (dropdown) => {
    this.setState({
      navDrop: this.state.navDrop === dropdown ? null : dropdown,
    });
  };

<NavDropdown
  className="navbar-nav active"
  title="About"
  id="navbarJoin"
  onMouseEnter={() => {
  this.handleToggle("first");
  }}
  onMouseLeave={() => {
    this.handleToggle("first");
  }}
  show={this.state.navDrop === "first"}
  >
</NavDropdown>
```

[the remark this answers is not in the document]
Comment (aayush-05) [reply src/components/NavBar/index.js]: Merged your PR.
Thanks for this suggestion; learnt a better way to do this :)

**Pull request merged**